source/docs/zero-to-robot/step-2/wpilib-setup.rst

```diff
@@ -11,7 +11,7 @@ Supported Operating Systems and Architectures:
 
 .. warning:: The following OSes are no longer supported: macOS 12 or earlier, Ubuntu 18.04 & 20.04, Windows 7, Windows 8.1, and any 32-bit Windows.
 
-.. note:: [Windows 10 support from Microsoft will end in October 2025](https://www.microsoft.com/en-us/windows/end-of-support). We intend to continue supporting Windows 10 through the 2026 season, but may have to drop support in 2027. Teams should start planning their upgrade path to Windows 11.
+.. note:: [Windows 10 support from Microsoft will end in October 2025](https://www.microsoft.com/en-us/windows/end-of-support). We intend to continue supporting Windows 10 through the 2026 season, but may have to drop support in 2027. Teams should start planning their upgrade path to Windows 11, or switch to one of the supported Linux distributions listed.
```

Comment:
I don't think switching to Linux is really a viable solution for most "zero to robot" teams.  Additionally support of components like the driver station are still unknown but switching a computer to Linux would have the additional downside of potentially not supporting the game tools.

Reply:
Some systems cannot be upgraded to 11 for one reason or another.  How about this? "Teams should start planning their upgrade path to Windows 11 if supported, or otherwise consider migrating to one of the supported Linux distributions listed."